cmd/ipfs/main_test.go

```diff
@@ -1,17 +1,20 @@
 package main
 
 import (
-	"testing"
-
 	"github.com/ipfs/go-ipfs/commands"
+
+	gc "gx/ipfs/QmPp8w6dCzx6vPNre8QVSbN7GCY9M7ornWesnrJg48vYxE/check.v1"
 )
 
-func TestIsCientErr(t *testing.T) {
-	t.Log("Catch both pointers and values")
-	if !isClientError(commands.Error{Code: commands.ErrClient}) {
-		t.Errorf("misidentified value")
-	}
-	if !isClientError(&commands.Error{Code: commands.ErrClient}) {
-		t.Errorf("misidentified pointer")
-	}
+func (m mainRun) TestIsCientErr(c *gc.C) {
```

Comment:
I have to agree with @whyrusleeping on this, in this case `misidentified value` is much more descriptive than `expected: true, got: false`.

Also replacing the `c.Assert(ok, gc.Equals, true)` with
```go
if !ok {
    t.Error("misidentified value")
}
```
Is easier and faster to type, conveys more information and everybody knowns what is going on.
There is no value added or `c.Assert` in this part of code.